catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/filter/VisitableJsonElementImpl.java

```diff
@@ -87,30 +81,41 @@
           .put(">=", FilterVisitor2::visitBinaryComparisonType)
           .put("<", FilterVisitor2::visitBinaryComparisonType)
           .put("<=", FilterVisitor2::visitBinaryComparisonType)
-          .put("LIKE", FilterVisitor2::visitPropertyIsLikeType)
           .put("ILIKE", FilterVisitor2::visitPropertyIsLikeType)
           .build();
 
   private final BiConsumer<FilterVisitor2, VisitableElement> visitMethod;
   private final String name;
   private final Object value;
 
+  private String functionName;
+
   public static VisitableJsonElementImpl create(final FilterNode node) {
     return new VisitableJsonElementImpl(node);
   }
 
-  private VisitableJsonElementImpl(String prop) {
-    this.visitMethod = FilterVisitor2::visitString;
+  /**
+   * Base constructor.
+   *
+   * @param visitMethod dispatch method for visiting.
+   * @param value value of this node.
+   */
+  private VisitableJsonElementImpl(
+      final BiConsumer<FilterVisitor2, VisitableElement> visitMethod, final Object value) {
     this.name = null;
-    this.value = prop;
+    this.functionName = null;
+
+    this.visitMethod = visitMethod;
+    this.value = value;
   }
 
+  /**
+   * Primary constructor for evaluating a {@link FilterNode} and constructing the visitable tree
+   * accordingly.
+   *
+   * @param node the node to construct the tree for.
+   */
   private VisitableJsonElementImpl(final FilterNode node) {
-    if (LIKE.equals(node.getOperator())) {
-      throw new UnsupportedOperationException("LIKE (case sensitive) currently is not supported");
-      // Ticket for adding support - https://codice.atlassian.net/browse/DDF-3829
```

Comment:
I'm guessing no one is tracking https://codice.atlassian.net/browse/DDF-3829 , so it probably doesn't matter, but it could be updated

Reply:
I'm also a little concerned about changing that ticket to "fixed" status. This fix works but ultimately we want whatever simpler system was constructed as part of the new UI to be used over this. New projects shouldn't leverage this existing solution, IMO.